pathgen/pathgen.go

```diff
@@ -171,10 +171,10 @@ type GenConfig struct {
 	GenerateWildcardPaths bool
 	// SplitByModule controls whether to generate a go package for each yang module.
 	SplitByModule bool
-	// TrimOCPackage controls whether to trim openconfig- from generated go package names.
-	TrimOCPackage bool
-	// BaseImportPath is used to create to full import path of the generated go packages.
-	BaseImportPath string
+	// TrimPackagePrefix is the prefix to trim from generated go package names.
+	TrimPackagePrefix string
```

Comment:
Likewise here I'd prefer a name like `TrimPackageOrgPrefix` and mention that the Go packages are split and named according to the module that instantiated the node.